auto-configurations/models/spring-ai-autoconfigure-model-ollama/src/main/java/org/springframework/ai/model/ollama/autoconfigure/OllamaApiAutoConfiguration.java

```diff
@@ -42,7 +42,7 @@ public class OllamaApiAutoConfiguration {
 
 	@Bean
 	@ConditionalOnMissingBean(OllamaConnectionDetails.class)
-	public PropertiesOllamaConnectionDetails ollamaConnectionDetails(OllamaConnectionProperties properties) {
```

Comment:
`PropertiesOllamaConnectionDetails` is a package-protected inner class that was marked as the return type from a public API method. Although the compiler works with that, the IDE throws a visibility issue warning.